Base.gitlab-ci.yml

```diff
@@ -70,7 +70,7 @@ default:
     CONFIGURATION_VERSION_ID: $configuration_version_id
     MESSAGE: "Base template message. Override this"
```

Comment:
Do we need to add `SAVE_PLAN` and `IS_DESTROY` here? or not necessary?

Reply:
I think we would need to do that if these variables were generated by intermediate stages or to override them. 
```
CONFIGURATION_VERSION_ID: $configuration_version_id
```
`$configuration_version_id` was set by an earlier stage which is why we have defined it here as a variable.
Are `SAVE_PLAN` and `IS_DESTROY` provided by the user directly?

Reply:
That is correct. So we should be good as is if I'm understanding you correctly.